src/tests/Directory.Build.props

```diff
@@ -129,6 +124,8 @@
     <!-- Disable some C# warnings for the tests. -->
     <NoWarn>78,162,164,168,169,219,251,252,414,429,618,642,649,652,659,675,1691,1717,1718,3001,3002,3003,3005,3008,8981</NoWarn>
     <RunAnalyzers>false</RunAnalyzers>
+    <EnableTrimAnalyzer>false</EnableTrimAnalyzer>
```

Comment:
I have tried to disable it in more fine grained way first, but it turned out to be whack-a-mole. 

This matches what we do for libraries tests.

Reply:
We could condition on EnableAggressiveTrimming. Those should be clean

Reply:
They are not clean today. I agree it would be a good path to take if we wanted to spend time on incrementally enabling analyzers for test build.